rules/macos/process_creation/proc_creation_macos_enable_root_account.yml

```diff
@@ -0,0 +1,41 @@
+title: Enable the Root Account
+id: 821bcf4d-46c7-4b87-bc57-9509d3ba7c11
+status: experimental
+description: Detects attempts to enable/add an account to the admin group, thus granting the root privilege
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1078.001/T1078.001.md
+    - https://github.com/elastic/detection-rules/blob/4312d8c9583be524578a14fe6295c3370b9a9307/rules/macos/persistence_enable_root_account.toml
+    - https://ss64.com/osx/dsenableroot.html
+    - https://ss64.com/osx/dseditgroup.html
+    - https://ss64.com/osx/dscl.html
+author: Sohan G (D4rkCiph3r)
+date: 2023/02/18
+tags:
+    - attack.t1078
+    - attack.t1078.001
+    - attack.initial_access
+    - attack.persistence
+logsource:
+    category: process_creation
+    product: macos
+detection:
+    selection1:
+        Image|endswith:
+            - '/dseditgroup'
+            - '/dscl'
+    selection2:
+        CommandLine|contains:
+            - '/Groups/admin'
```

Comment:
'/Groups/admin' will be cover by 'admin'

Reply:
Updated the rule!